ydb/docs/en/core/reference/docker/environment.md

```diff
@@ -0,0 +1,31 @@
+# Environment variables for running {{ ydb-short-name }} in Docker
```

Comment:
Let's rename the file to something along the lines of configuration.md, as the usage of environment variables is more like an implementation detail. Also, sooner or later someone would want to use an arbitrary config file, and the environment variables won't be the only option.